### DISCUSSION TITLE
Possible to forecast with AutoARIMA in this case?

### DISCUSSION THREAD

**Discussion by matsuobasho** [Nixtla/statsforecast#972] [Q&A]
I have a time series where I'm trying to predict the sales for many different category-subcategory combinations.  I want the model to learn across these combinations, so from my understanding, I shouldn't specify a unique_id for every separate combination, because then I would be building a separate model for each.  

If this is correct, here's my actual question.
I need to split the train test by merchants, not by time.  So in essence my training set contains the data for the train merchants.  My forecast horizon is 365 days.  Currently, when I feed the test set:
`fcst = sf.forecast(df=train, h=horizon, X_df=test)`

I get a ValueError that a 365 row dataframe is expected.  

Note the exogenous variables - these would be categorical variables I engineered.  

I understand that I can potentially just use an XGBoost model and create my own lags on the y variable.  However, I'd like to see Arima's performance on this.  Is there a way to structure the test set so that this would work?  Ultimately, for prediction, I want to provide one day's worth of data for each test category subcategory, as well as the exogenous variables for the next 365 days, and then have the model predict in an autoregressive manner.

**Comment by jmoralez** [comment]
Hey. The ARIMA model is designed to work on a single serie, so I don't think you'd get good results with this. Another possible approach would be forecasting the aggregate and then reconciel with our [hierarchicalforecast](https://github.com/Nixtla/hierarchicalforecast/) package.
